db/schema.rb

```diff
@@ -31,6 +31,7 @@
   create_table "email_templates", force: :cascade do |t|
     t.string   "title"
     t.string   "subject"
+    t.string   "from"
```

Comment:
You did not add this field via migrations, so it should not be here.
If you are adding a migration - pay attention to changes in the schema.rb